### PR TITLE
Add setPotionUseTimeLeft to Witch

### PR DESCRIPTION
**Pull request by Owen1212055**
It seems to have had the method already in EntityWitch, perhaps it was forgotten?

### CODE REVIEW

**Review by kennytv — CHANGES_REQUESTED** [review]
Add a note on get/set that this only has an effect if the Witch is actually drinking (checked via `isDrinkingPotion`)

File: Spigot-API-Patches/0161-Add-more-Witch-API.patch [hunk 1]
```diff
@@ -37,6 +37,15 @@ index aa88aede6c4e66a608a63d07bc66d60357b0bee9..cd8d0a20e52ad35f659175e0d24a1dc0
 +    int getPotionUseTimeLeft();
 +
 +    /**
++     * Set time remaining (in ticks) that the Witch is drinking a potion
++     * This only has an affect while the Witch is drinking a potion.
++     * See {@link #isDrinkingPotion}
++     *
++     * @param ticks Time remaining
```
Comment (Proximyst) [inline Spigot-API-Patches/0161-Add-more-Witch-API.patch]:
```suggestion
+     * Set time remaining (in ticks) that the Witch is drinking a potion.
+     * <p>
+     * This only has an effect while the Witch is drinking a potion.
+     *
+     * @param ticks Time in ticks remaining
+     * @see #isDrinkingPotion
```

**Review by Proximyst — DISMISSED** [review]

File: Spigot-API-Patches/0161-Add-more-Witch-API.patch (hunk 1 above)
Comment (Owen1212055) [reply Spigot-API-Patches/0161-Add-more-Witch-API.patch]: thanks

**Comment by kennytv** [comment]
Committing multi line suggestions on a patch file (⓿_⓿)

**Comment by Owen1212055** [comment]
> Committing multi line suggestions on a patch file (⓿_⓿)

Ahh that would make sense, I was confused why when I did it before with someone it worked fine. Thanks for the info!

File: Spigot-API-Patches/0161-Add-more-Witch-API.patch [hunk 2]
```diff
@@ -37,6 +37,16 @@ index aa88aede6c4e66a608a63d07bc66d60357b0bee9..cd8d0a20e52ad35f659175e0d24a1dc0
 +    int getPotionUseTimeLeft();
 +
 +    /**
++     * Set time remaining (in ticks) that the Witch is drinking a potion.
++     * <p>
```
Comment (me4502) [inline Spigot-API-Patches/0161-Add-more-Witch-API.patch:41]: Where's the closing tag?

Comment (Proximyst) [reply Spigot-API-Patches/0161-Add-more-Witch-API.patch:41]: Missing close tag is legal (& standard-ish) in Javadocs. Don't ask why, though :p

**Review by MiniDigger — APPROVED** [review]

**Review by kennytv — APPROVED** [review]
Thank you

**Pull request merged**